python/README.md

```diff
@@ -0,0 +1,64 @@
+# Installation
+
+To use the `aida_interchange` package within your project you must first install it using the instructions described in the [Install setup.py](install-setup.py) section. It is recommended that you install `aida_interchange` into a virtual environment for development purposes. To install `aida_interchange`  within your virtual environment first activate your virtual environment and run the install command below. See [Python Development Setup](python-development-setup) for more details on creating a virtual environment.
```

Comment:
If we have to set up / activate our virtual environment before we can install setup.py, couldn't we just organize the README in this order?  That is, something like this:

- Python Development Setup
-- Install virtualenv
-- Create virtual environment
-- Activate virtual environment
-- Deactivate virtual environment
- Installation
-- Install setup.py
- Documentation

Reply:
Yes, the thought was that most people working with python are familiar with creating a virtual environment and will not need these instructions. I figured the actual installation was more important and should be at the top. I am open to either way.

Reply:
Sounds reasonable...I don't feel strongly either way.